apps/admin_panel/assets/src/omg-invite-modal/index.js

```diff
@@ -61,35 +65,112 @@ const InviteButton = styled(Button)`
   padding-left: 40px;
   padding-right: 40px;
 `
-export default class InviteModal extends Component {
+const enhance = compose(withRouter, connect(null, { inviteMember, getListMembers }))
+class InviteModal extends Component {
   static propTypes = {
     open: PropTypes.bool,
-    onRequestClose: PropTypes.func
+    onRequestClose: PropTypes.func,
+    inviteMember: PropTypes.func.isRequired,
+    getListMembers: PropTypes.func.isRequired,
+    match: PropTypes.object,
+    location: PropTypes.object
+  }
+  state = {
+    email: null,
+    role: 'viewer',
+    submitStatus: null
+  }
+  validateEmail = email => {
+    return /@/.test(email) && email.length !== 0
```

Comment:
I think `&& email.length !== 0` is redundant but checking for "@" sign is fine. We discussed before that this is enough for input form email validation, actual validation will always be done by sending an actual email to confirm.